cmd/main.go

```diff
@@ -145,6 +147,13 @@ func main() {
 
 	var openshiftVersionsMap models.OpenshiftVersions
 
+	if !isJSONString(Options.OpenshiftVersions) {
+		content, err := ioutil.ReadFile(ocp_version_file)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
I think that log should contain the message: "Failed to read default OCP version file "

Reply:
please use `failOnError`

Reply:
not sure if there is an option to failOnError and return value, from the examples on main I see we don't really use this format in those cases.
@filanov is that an option somehow?

Reply:
I don't understand the question, you don't return anything here, once you call `log.Fatal` the program will exit.